### PR TITLE
Add a setting for vibration

### PR DESCRIPTION
**Pull request by bramkragten**

### CODE REVIEW

File: src/fake_data/provide_hass.ts [hunk 1]
```diff
@@ -178,6 +178,7 @@ export const provideHass = (
 
     translationMetadata: translationMetadata as any,
     dockedSidebar: "auto",
+    vibrate: true,
```
Comment (balloob) [inline src/fake_data/provide_hass.ts]: Let's not put this in `hass` but store this in `coreUserData` instead like we do for advanced mode

Comment (balloob) [reply src/fake_data/provide_hass.ts]: oh interesting, maybe you are right with this approach, as with this approach it's a per-browser setting and not per-user.

Comment (bramkragten) [reply src/fake_data/provide_hass.ts]: Yeah, that was the idea

Comment (balloob) [reply src/fake_data/provide_hass.ts]: ok, great

File: src/translations/en.json [hunk 2]
```diff
@@ -1214,6 +1214,10 @@
           "header": "Always hide the sidebar",
           "description": "This will hide the sidebar by default, similar to the mobile experience."
         },
+        "vibrate": {
+          "header": "Vibrate",
+          "description": "Enable or disable vibration on this device."
```
Comment (balloob) [inline src/translations/en.json]: Maybe mention when vibrations could happen? "Enable or disable vibration on this device when controlling devices."

**Review by balloob — APPROVED** [review]

**Pull request merged**